src/components/biggive-form-field-select/biggive-form-field-select.tsx

```diff
@@ -71,15 +70,37 @@ export class BiggiveFormFieldSelect {
   render() {
     const greyIfRequired = this.backgroundColour === 'grey' ? ' grey' : '';
 
+    let options: Record<string, string>;
+    if (typeof this.options === 'string') {
+      options = JSON.parse(this.options);
+    } else {
+      options = this.options;
+    }
+    if (Array.isArray(options)) {
+      options = Object.fromEntries(options.map((value: string) => [value, value]));
+    }
+
+    if (typeof this.placeholder === 'string' && !this.placeHolderRemoved) {
+      options = Object.assign({ __placeholder__: this.placeholder }, options);
+    }
+
     return (
-      <div class={'dropdown space-below-' + this.spaceBelow + ' select-style-' + this.selectStyle + (this.prompt === null ? '  noprompt' : '')}>
-        <div class="sleeve" onClick={this.toggleFocus} onMouseLeave={this.toggleFocus}>
-          <span class={'placeholder' + greyIfRequired}>{this.selectedLabel === null || this.selectedLabel === undefined ? this.placeholder : this.selectedLabel}</span>
-        </div>
-        <div class={'options' + greyIfRequired}>
-          <slot></slot>
-        </div>
-        {this.prompt && <div class={'prompt' + greyIfRequired}>{this.prompt}</div>}
+      <div>
+        <label class={greyIfRequired}>
```

Comment:
@bdsl Given there's nesting around both the label text and the element itself, I think it might be better to explicitly associate the `<label/>`. Even in simpler markup scenarios,
> Generally, explicit labels are better supported by assistive technology.

– https://www.w3.org/WAI/tutorials/forms/labels/#associating-labels-implicitly